app/build.gradle

```diff
@@ -48,6 +48,27 @@ android {
         }
     }
 
+    flavorDimensions "stage"
+    productFlavors {
+        production {
+            applicationId "co.nimblehq"
+        }
+
+        staging {
```

Comment:
Why do we need a staging variants I think production and develop are cover the product flavor that we have  🤔 What do you think?

Reply:
@Wadeewee I think we should keep them in template because they are typical variants for development process, later we can update or remove variants based on each project's requirement 🙏

Reply:
Just curious about Omise & Deemoney project, what variants are you guys using in the project?

Reply:
@Tuubz on DeeMoney, we have `staging` and `production`. cc @luongvo ☝️

Reply:
Yeah, in 7-Eleven we only have two variants as well. Which makes me think it makes more sense to have only 2 instead of 3 🙏  Let's see with Omise team says

Reply:
actually, development and staging are quite similar, we have staging, uat and production in Omise project
I think we could combine dev and staging @manh-t